helm/g2p-sandbox/values.yaml

```diff
@@ -143,7 +143,7 @@ ph-ee-engine:
 
   ph_ee_connector_ams_mifos:
     enabled: true
-    image: docker.io/openmf/ph-ee-connector-ams-mifos:v1.4.0
+    image: docker.io/openmf/ph-ee-connector-ams-mifos:latest
```

Comment:
this cant be latest, in a release all the image versions should be pointing to a tagged version